src/main/java/com/svydovets/bibirnate/session/Session.java

```diff
@@ -1,5 +1,16 @@
 package com.svydovets.bibirnate.session;
 
+/**
+ * This is main interface to work with Bibirnate. It allows to read operations for mapped classes.
+ */
```

Comment:
not only read operations, also persist, delete

Reply:
For now, it is only read) When we add methods to persist and delete we will update the docs.